### ISSUE TITLE
ERROR:  duplicate key value violates unique constraint "xpk_measurement"

### ISSUE THREAD

**Issue by smondet** [MIT-LCP/mimic-omop#49]
After trying the fix for #48, I got:

```
...
INSERT 0 11656610
Time: 1585618.783 ms
INSERT 0 630856
Time: 81014.760 ms
psql:etl/StandardizedClinicalDataTables/MEASUREMENT/etl.sql:804: ERROR:  duplicate key value violates unique constraint "xpk_measurement"
DETAIL:  Key (measurement_id)=(301930857) already exists.
Time: 5491441.549 ms
```

**Comment by parisni** [comment]
@smondet  thanks for reporting those. But remember, you should not add the indexes before loading the tables. The ETL might be far longer in this way. I suggest you to remove them. (updating indexes way more complex than building them from scratch)

**Comment by spfohl** [comment]
Do you mean that we should not build the indexes with "psql "$OMOP" -f "omop/build-omop/postgresql/OMOP CDM postgresql indexes.txt"" until after running the ETL with "psql "$MIMIC" -f "etl/etl.sql""? I'm just trying to clarify because the order implied by the README suggests that we should build the indexes first.

**Comment by alistairewj** [comment]
That was probably written by me - and probably it is a lot slower than it could be by doing it in my suggested order!

**Comment by parisni** [comment]
Yes exactly: indexes, integrity constraints shall be added after the ETL step.

Let me modify the REAME accordingly.

**Comment by parisni** [comment]
well  @alistairewj  looks ready to update his great documentation

**Comment by spfohl** [comment]
Awesome, that's good to know. I have also been having similar issues with duplicate keys, but I'll try re-running with the indexes at the end.
